pallets/ds-accounts/src/lib.rs

```diff
@@ -209,6 +224,28 @@ decl_module! {
             Ok(())
         }
 
+        #[weight = <T as Trait>::WeightInfo::register_pilot()]
+        pub fn register_pilot(origin, account: T::AccountId) -> dispatch::DispatchResult {
+            // Check that the extrinsic was signed and get the signer.
+            // This function will return an error if the extrinsic is not signed.
+            // https://substrate.dev/docs/en/knowledgebase/runtime/origin
+            let who = ensure_signed(origin)?;
+            ensure!(Self::account_is_registrar(&who), Error::<T>::NotAuthorized);
+
+            // Update storage.
+            AccountRegistry::<T>::mutate(&account, |acc|{
+                debug::info!("register_pilot: roles:{:?} create_time={:?}", acc.roles, acc.create_time);
```

Comment:
Critical security error!  Any registrar can disable the admin and others accounts making them pilots